README.md

```diff
@@ -26,17 +26,18 @@ When **creating a new major branch for a pre-release major version**
 
 - Make sure you've added a new major branch to both `silverstripe/developer-docs` and `silverstripe/silverstripe-userhelp-content`
 - Add the new major to `sources-docs.js` and `sources-user.js`
-- Add new major branches for various modules as defined in `sources-docs.js` and `sources-user.js` as well
+- Add the new major branches for various modules as defined in `sources-docs.js` and `sources-user.js` as well
 - Add the new major to the version select in `src/components/Header.tsx`
-- Add the major to the `PRE_RELEASE` array in the `getVersionMessage` function in `src/utils/nodes.ts`
+- Add the new major to the `PRE_RELEASE` array in the `getVersionMessage` function in `src/utils/nodes.ts`
+- Add the new major to `redirects` in `gatsby-node.js`
```

Comment:
This step was missing and is necessary for e.g. `https://docs.silverstripe.org/en/getting_started/environment_management/` to get redirected to `https://docs.silverstripe.org/en/6/getting_started/environment_management/`